Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.8-alpine
```

Comment:
I'm fine optimizing this to the broader community, and putting some burden on 18F to override the Dockerfile in its own implementation. We could also potentially have the Dockerfile use one image or another based on a CLI flag or env var.